csharp/src/IceBox/msbuild/icebox/net5.0/icebox.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../../../../msbuild/ice.common.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AssemblyName>iceboxnet</AssemblyName>
+    <Version>$(IceVersion)</Version>
+    <OutputPath>../../../../../bin</OutputPath>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup>
+    <UseAppHost>true</UseAppHost>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="../../../AssemblyInfoExe.cs" />
+    <Compile Include="../../../Server.cs" />
+    <Compile Include="../../../ServiceManagerI.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="../../../../Ice//msbuild/net5.0//ice.csproj" />
```

Comment:
Why all these `//`?

Reply:
probably an error with the sed command I use to rewrite the references, I will fix it

Reply:
Fixed